docs/changes.rst

```diff
@@ -37,6 +37,20 @@ Released: not yet
   surrounded by verbose test (if p.parse.verbose:). See pywbemcli issue
   #395,
 
+* Fixes bugs in SubscriptionManager where 1. host_name  of
+  CIM_ListenerDestination and CIM_IndicationFilter objects inconsistently used
+  in building the string in the Name property and building the regex to
+  find the string. Starting a subscription manager against a server with owned
+  subscriptions, etc. did not restore them as owned, 2. add_filter(),
+  add_destination() and add_subscriptions() did not create a complete path for
+  the instance to be added to be added. While this is correct for the
+  CreateInstance(), that path is used to determine if the instance already
+  exists a few lines further down in the code. 3. The client host fqdn was being
+  used as the instance SystemName property when the local instances were built.
+  WBEM servers consistently overwrite this property if is is incorrect or not
+  defined. The SystemName returned from the add_operations is now used to
```

Comment:
If WBEM servers consistently overwrite this property, it is not an issue, but it becomes an issue for those servers that don't. That is what we should state here.